actionview/lib/action_view/helpers/active_model_helper.rb

```diff
@@ -10,6 +10,23 @@ module ActiveModelHelper
 
     # = Active \Model Instance Tag \Helpers
     module ActiveModelInstanceTag
+      class TagBuilder
```

Comment:
This is a smell. Needing this proxy object points to me that isn't worthy doing this internal change. Maybe externally, yes, but internally the code is less direct and probably slower since we need to build a lot of object we didn't before.

Reply:
@rafaelfranca I've removed the reliance on method missing in favor of a `TagBuilder` class that inherits from the other to wrap its tags with error content.